locales/en.yml

```diff
@@ -19,9 +19,143 @@ events:
 posts:
   add: Add post
   title: Feed
+profile:
+  username:
+    label: Username
+    before: Use only letters, numbers, underscores and periods.
 profiles:
+  community: I dance in
+  type: Type
+  name:
```

Comment:
I mean all the fields, not only `username` 😅
Just like it was before your change

Reply:
Sorry, @razbakov  what does it mean?

Reply:
it's continuation of https://github.com/we-dance/platform/pull/114#discussion_r794848735